hardhat.config.js

```diff
@@ -146,11 +173,11 @@ console.log(
 (async (addressFrom, addressTo) => {
   console.log(
     `address from = ${addressFrom} balance=`,
-    await web3.eth.getBalance(addressFrom)
+    await getBalance(addressFrom)
```

Comment:
This is only to display the balance increased by the call to faucet (and optionally via call to getBalance)

Reply:
yes, but if web3.eth.getBalance failed, all our process failed too